modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/metrics/Metrics.scala

```diff
@@ -49,6 +49,11 @@ object Metrics {
       val value = v.toString
       val metricType = MetricType.Count
     }
+    final case class CountBad(v: Long) extends KVMetric {
```

Comment:
I'm not 100% comfortable with the loader reporting on bad rows.  Because the loader didn't really handle the bad row; it was handled entirely by the transformer.

But I understand that it must be done this way, because the batch transformer does not send statsd metrics.

So I don't have any better suggestion.  I think this is good as a pragmatic way to give us the visibility we need of this metric.